completions/zsh/_bwrap

```diff
@@ -65,6 +65,7 @@ _bwrap_args=(
     '--size[Set size in bytes for next action argument]: :->after_size'
     '--symlink[Create symlink at DEST with target SRC]:symlink target:_files:symlink to create:_files:'
     '--sync-fd[Keep this fd open while sandbox is running]: :_guard "[0-9]#" "file descriptor to keep open"'
+    '--sys[Mount new sysfs on DEST]:mount point for sysfs:_files -/'
```

Comment:
Perhaps

```suggestion
    '--sys[Mount new sysfs instance on DEST]:mount point for sysfs:_files -/'
```

to emphasize that, like `--proc`, this is not the same thing as bind-mounting the existing `/sys`.